test/cpp/util/cli_call.cc

```diff
@@ -61,6 +62,19 @@ CliCall::CliCall(const std::shared_ptr<grpc::Channel>& channel,
       ctx_.AddMetadata(iter->first, iter->second);
     }
   }
+
+  // Set deadline if timeout > 0 (default value -1 if no timeout specified)
+  if (FLAGS_timeout > 0) {
+    int64_t timeout_in_us = ceil(FLAGS_timeout * 1e6);
```

Comment:
Seems like since you're multiplying by 1e3 again, maybe just multiple by 1e9 and use timeout_in_nanos?

Reply:
Done.